code/game/area/Space_Station_13_areas.dm

```diff
@@ -621,6 +621,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	lighting_colour_bulb = "#ffebc1"
 	sound_environment = SOUND_AREA_WOODFLOOR
 
+/area/crew_quarters/bar/bar_spawn_area
```

Comment:
Won't this cause the bar to be renamed to Bar Spawn Area?
Currently the area remains the bar, as the spawners themselves have area pass-through.

Reply:
I see what you mean for the bar as you put pass-throughs for it